src/neynar-api/v2/client.ts

```diff
@@ -1347,6 +1349,7 @@ cursor?: string
       parentUrl?: string;
       channelId?: string;
       embedUrl?: string;
+      embedTypes?: EmbedType[];
```

Comment:
Here we can accept it as an array and convert it into a comma separated string before passing it on